kernel-info.mk.example

```diff
@@ -198,14 +201,24 @@ BUILD_LLVM = 0
 # Set to 1 to skip modules packaging if CONFIG_MODULES is disabled in defconfig 
 BUILD_SKIP_MODULES = 0
 
+# Set clang version
+CLANG_VERSION = 6.0-4691093
+# Set to 1 to use a manually installed toolchain
+# Remember to update the path in BUILD_PATH
+CLANG_CUSTOM = 0
+
 # Extra paths to prepend to the PATH variable. You'll probably want
 # to specify the clang path here (the default).
-BUILD_PATH = /usr/lib/llvm-android-6.0-4691093/bin
+BUILD_PATH = /usr/lib/llvm-android-$(CLANG_VERSION)/bin
 
 # Extra packages to add to the Build-Depends section. Mainline builds
 # can have this section empty, unless cross-building.
 # The default is enough to install the Android toolchain, including clang.
-DEB_TOOLCHAIN = linux-initramfs-halium-generic:arm64, binutils-aarch64-linux-gnu, clang-android-6.0-4691093, gcc-4.9-aarch64-linux-android, g++-4.9-aarch64-linux-android, libgcc-4.9-dev-aarch64-linux-android-cross
+DEB_TOOLCHAIN = linux-initramfs-halium-generic:arm64, binutils-aarch64-linux-gnu, gcc-4.9-aarch64-linux-android, g++-4.9-aarch64-linux-android, libgcc-4.9-dev-aarch64-linux-android-cross,
+# When using the Droidian clang, add to the build deps
+ifneq ($(CLANG_CUSTOM), 1)
```

Comment:
I think this can be moved to the actual kernel-snippet right?

Reply:
Yes, that way we would have a cleaner kernel-info.
Are you referring to DEB_TOOLCHAIN as well?

Reply:
Heh, DEB_TOOLCHAIN might be a bit more troublesome even if I'd like to hide it as well. But is used to add some other packages on top (some kernel packages for example build-depend on another version of the initramfs).

I would like to see it gone, but I'm okay to keep it around for now... only thing is that when we will add the clang package (if CLANG_CUSTOM is not used), we would probably need to handle cases where the original DEB_TOOLCHAIN misses a comma, as that might happen frequently

Reply:
I agree to keep the DEB_TOOLCHAIN in kernel-info, is useful. Was confused because the above mention. Sorry!

About the CLANG_CUSTOM, you are right.
What do you think about this?

DEB_TOOLCHAIN = linux-initramfs-halium-generic:arm64, binutils-aarch64-linux-gnu, gcc-4.9-aarch64-linux-android, g++-4.9-aarch64-linux-android, libgcc-4.9-dev-aarch64-linux-android-cross

ifneq ($(CLANG_CUSTOM), 1)
DEB_TOOLCHAIN := clang-android-$(CLANG_VERSION), $(DEB_TOOLCHAIN)
endif

Reply:
I pushed to my sid new code with both suggestions: move the CLANG_CUSTOM condition to kernel-snippet and fix the comma problem.

Take a look when you can, thank you!

Reply:
thanks a lot, looks great!